client/src/components/CreateAccount/CreateAccount.module.scss

```diff
@@ -0,0 +1,32 @@
+.CreateAccount {
+  &__TopBar {
+    // Must be !important, otherwise styling doesn't work
+    background: transparent !important;
+    box-shadow: none !important;
+    color: black !important;
+  }
+
+  &__CenteredTypography {
+    display: flex;
+    flex-grow: 1;
+    justify-content: center;
+  }
+
+  &__Container {
+    display: flex;
+    flex-direction: column;
+    align-items: center; /* Center horizontally */
```

Comment:
Just as a fun fact, Sass lets you use traditional `//` comments instead of `/**/` ones. Not saying you should change these, but it's good to know for future reference 🔥

Reply:
Thanks for letting me know! I did just copy these from GeneralInfo.module.scss though.